media/windows/ECC-MQTT-IoT/ECC_MQTT_IoT_SQLite.py

```diff
@@ -963,7 +1128,13 @@ def send_mail(mail_origin,
     mail_time_str = mail_time_str + " -" + f'{(time.timezone / 3600):02.0f}00'
 
     try:
+<<<<<<< Updated upstream
```

Comment:
Looks like a conflict resolution got accidentally left in here.

Reply:
Yes, I noticed this also and hopefully corrected it now...